<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>441</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Додати Комплектуючий</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Материнська плата</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Процесор</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Відеокарта</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Модуль пам'яті</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HDD/SSD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Блок живлення</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Охолодження</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Корпус</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Назва:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_line"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ціна:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cost_line"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="text"/>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Додати</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
